lib/src/widgets/chat/video/ds_video_message_bubble.widget.dart

```diff
@@ -83,6 +87,7 @@ class _DSVideoMessageBubbleState extends State<DSVideoMessageBubble>
       url: widget.url,
       mediaSize: widget.mediaSize,
       httpHeaders: widget.shouldAuthenticate ? DSAuthService.httpHeaders : null,
+      fileName: widget.title ?? '',
```

Comment:
What happens if title is null?

Reply:
This only occurs when the video is sent to the app. When you download the video, it will be saved locally using the uniqueId as the title. Next time, it will search for the title and find the video.